RUNBOOK.md

```
@@ -644,3 +645,39 @@ gh run list --workflow claude-security-review.yml --repo mark-brannan/dotfiles -
+
+## A dev server in WSL2 is unreachable from any other device
+
+A server started inside WSL2 answers on every address from inside WSL —
+loopback, LAN, Tailscale — and times out from a phone, a tablet, or even the
+Windows host it is running on. Nothing is wrong with the server. Under
+mirrored networking WSL shares the Windows network namespace, so Windows
+Firewall governs its inbound traffic and blocks it by default.
+
+From the Windows host itself, `localhost` works with no change:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)